addons/mail/static/src/components/composer_text_input/composer_text_input.xml

```diff
@@ -36,7 +36,7 @@
                         </div>
                     </div>
                 </t>
-                <textarea class="o_ComposerTextInput_textarea" t-esc="composer.textInputContent" t-att-placeholder="textareaPlaceholder" t-on-focusin="_onFocusinTextarea" t-on-focusout="_onFocusoutTextarea" t-on-keydown="_onKeydownTextarea" t-on-keyup="_onKeyupTextarea" t-on-input="_onInputTextarea" t-ref="textarea"/>
+                <textarea class="o_ComposerTextInput_textarea" t-att-class="{ 'o-composer-is-compact': props.isCompact }" t-esc="composer.textInputContent" t-att-placeholder="textareaPlaceholder" t-on-focusin="_onFocusinTextarea" t-on-focusout="_onFocusoutTextarea" t-on-keydown="_onKeydownTextarea" t-on-keyup="_onKeyupTextarea" t-on-input="_onInputTextarea" t-ref="textarea"/>
```

Comment:
I think we should change the behavior of the text input shortcuts.

When we have an extended composer (mailing channel), I think the Enter key should go to a new line just like in chatter.

Both because a mistake quickly happens and we wouldn't want to send a email to tons of people, and because it is usual to go to a new line when sending an email.

(commenting here because there is a diff but change should be in `_onKeydownTextareaEnter` itself)